controller/src/main/java/io/pravega/controller/store/task/InMemoryTaskMetadataStore.java

```diff
@@ -32,13 +30,9 @@
     @GuardedBy("itself")
     private final Map<Resource, LockData> lockTable;
 
-    @GuardedBy("itself")
-    private final Map<String, Set<TaggedResource>> hostTable;
-
     InMemoryTaskMetadataStore(ScheduledExecutorService executor) {
-        super(executor);
+        super(new InMemoryHostIndex(), executor);
         lockTable = new HashMap<>();
```

Comment:
Looks like we could use these methods. Filed an issue #1423 to explore that possibility and simplify implementation.